lib/actions/request_review.js

```diff
@@ -0,0 +1,42 @@
+const { Action } = require('./action')
+const _ = require('lodash')
+
+const createRequestReview = async (context, number, reviewers) => {
+  return context.github.pullRequests.createReviewRequest(
+    context.repo({ number, reviewers })
+  )
+}
+
+class Comment extends Action {
```

Comment:
Class name should be `RequestReview` here. :-)

Reply:
Done